### PR TITLE
GL-103: Add Admin UI for creating category assessment data

### PR DESCRIPTION
**Pull request by rasikasri**
### Jira link

[GL-103](https://transformuk.atlassian.net/browse/GL-103)

### What?

I have added/removed/altered:

- [x] Added a form to create category assessment record
- [x] Added selection of theme to associate with new category assessment
- [x] Removed service switching link from category assessment page

### Why?

I am doing this because:

- Admin user should be able to create operations on the categorisation data for the Green Lanes
-
-

### Have you? (optional)

- [ ] Reviewed view changes with stake holders
- [ ] Validated mobile responsive behaviour of view changes

### CODE REVIEW

File: app/models/green_lanes/theme.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    attributes :section,
+               :subsection,
+               :theme,
+               :description,
+               :category
+
+    collection_path '/admin/green_lanes/themes'
+
+    def label
+      formatted_label.length > MAX_LENGTH ? "#{formatted_label[0...MAX_LENGTH]}..." : formatted_label
```
Comment (jebw) [inline app/models/green_lanes/theme.rb:17]: Rails has a `truncate` method on strings which will do this for you

```ruby
formatted_label.truncate(MAX_LENGTH)
```

Comment (jebw) [reply app/models/green_lanes/theme.rb:17]: https://api.rubyonrails.org/classes/String.html#method-i-truncate

**Review by jebw — APPROVED** [review]
LGTM - added suggestion to use the built in truncate method ?

File: app/models/green_lanes/theme.rb (hunk 1 above)
Comment (rasikasri) [reply app/models/green_lanes/theme.rb]: fixed

**Review by bhavanibandla — APPROVED** [review]
Approved this PR to merge into the main branch

**Pull request merged**